mempool/reactor.go

```diff
@@ -31,13 +31,25 @@ const (
 	maxActiveIDs = math.MaxUint16
 )
 
+// MempoolWithWait extends the standard Mempool interface to allow reactor to
+// wait for transactions and iterate on them once there are any. Also it
+// includes ReapMaxTxs function, which is useful for testing.
+//
+// UNSTABLE
+type MempoolWithWait interface {
```

Comment:
better name?

Reply:
https://github.com/tendermint/tendermint/issues/2659#issuecomment-479084707

Reply:
> better name?

`WithWait`. If you use it somewhere, it would be `mempool.WithWait`then.